app.json

```diff
@@ -11,6 +11,9 @@
     {
       "url": "https://github.com/heroku/heroku-buildpack-nodejs"
     },
+    {
+      "url": "https://github.com/moneymeets/python-poetry-buildpack"
+    },
```

Comment:
Heroku's Python buildpack only supports pip. The poetry buildpack basically creates a requirements.txt file and a runtime.txt file based on pyproject.toml and poetry.lock, then lets the normal Python buildpack do its thing.